install_config/router/default_haproxy_router.adoc

```diff
@@ -907,6 +907,195 @@ This example uses `curl` with a local resolver to simulate the DNS lookup:
 # curl -k --resolve anyname.example.test:443:$routerip https://anyname.example.test/
 ----
 
+[NOTE]
+====
+For routers that allow wildcard routes (`ROUTER_ALLOW_WILDCARD_ROUTES`
+set to `true`), there are some caveats to the ownership of a subdomain
+associated with a wildcard route.
+
+Prior to wildcard routes, ownership was based on the claims made for a
+host name with the oldest route winning against any other claimants.
+E.g. route `r1` in namespace `ns1` with a claim for `one.example.test`
+     would win over another route `r2` in namespace `ns2` for the same
+     host name `one.example.test` if route `r1` was older than route `r2`.
+
+In addition, routes in other namespaces were allowed to claim
+non-overlapping hostnames.
+E.g. route `rone` in namespace `ns1` could claim `www.example.test` and
+   another route `rtwo` in namespace `d2` could claim `c3po.example.test`).
+
+This is still the case if there are _NO_ wildcard routes claiming that same
+subdomain (`*.example.test` in the above example).
```

Comment:
Do we want to say *. there?  The subdomain is just example.test.

Or am I being too pedantic?